src/xrc/src/periodic.rs

```diff
@@ -290,16 +291,20 @@ mod test {
             "EUR".to_string() => 10_000,
             "SGD".to_string() => 1_000,
             "CHF".to_string() => 7_000,
+            COMPUTED_XDR_SYMBOL.to_string() => 10_000,
         };
-        let mock_forex_sources = MockForexSourcesImpl::new(vec![map], vec![]);
+        // The [ForexRatesStore] would only return a value if it has at least 4 sources for CXDR
+        // So we add here four sources with the same rates.
```

Comment:
If we increase the constant later, this piece of code needs to be updated. We could just say here that we add "sufficiently many sources" so that we don't have to update the comment as well.   :-)